src/vue-rut.js

```diff
@@ -42,12 +42,14 @@ export function rutFilter(value) {
 
 export const rutInputDirective = {
   bind() {
-    this.el.addEventListener('blur', (event) => {
-      event.target.value = rutFilter(event.target.value) || '';
+    const event = (this.arg === 'live') ? 'input' : 'blur';
```

Comment:
'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).